src/tests/functional/plugin/gpu/shared_tests_instances/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -142,36 +128,24 @@ INSTANTIATE_TEST_SUITE_P(smoke_MultiCompileModelBehaviorTests,
 const std::vector<ov::AnyMap> auto_setcore_properties = {
     {ov::device::priorities(CommonTestUtils::DEVICE_GPU),
      ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT),
-     ov::hint::num_requests(2),
-     ov::hint::allow_auto_batching(false),
-     ov::enable_profiling(false)},
```

Comment:
1) here delete allow_auto_batch, but shouldn' this go together with PR removal of batch keys from auto list?
2) enable_profiling is removed , why?